libs/partners/prompty/langchain_prompty/renderers.py

```diff
@@ -0,0 +1,15 @@
+from pydantic import BaseModel
+from .core import Invoker, InvokerFactory, Prompty, SimpleModel
+import chevron
+
+class MustacheRenderer(Invoker):
+    def __init__(self, prompty: Prompty) -> None:
+        self.prompty = prompty
+
+    def invoke(self, data: BaseModel) -> BaseModel:
+        assert isinstance(data, SimpleModel)
+        generated = chevron.render(self.prompty.content, data.item)
```

Comment:
this will actually be a security issue too because of chevron partials. Would be better if you just constructed a `ChatPromptTemplate` with type as mustache!

As an alternative, you can also use the partial-free renderer in `langchain_core` that mustache `ChatPromptTemplate`s rely on https://github.com/langchain-ai/langchain/pull/19980/files#diff-c591786bc4be9ea7fab44200ea051b2e29ad2cda359a8cd3577eac4d0b2b0c7bR386

Reply:
ah nice! Didn't know you have this, switching to your partial free renderer now